extractor/filesystem/language/javascript/packagelockjson/extractor.go

```diff
@@ -68,22 +267,22 @@ func New(cfg Config) *Extractor {
 	}
 }
 
-// Name of the extractor.
```

Comment:
The reason for those sentence structure in comments is this style guide decision:
https://google.github.io/styleguide/go/decisions.html#comment-sentences

Reply:
Done